app/api/auth/[...nextauth]/options.ts

```diff
@@ -42,9 +42,17 @@ async function newClient(credentials: { host: string, port: string, password: st
         }
     });
 
-    // Verify connection
-    await client.connection.ping()
-    return client
+    // Verify connection and Role
+    const admin = await client.connection.aclGetUser(credentials.username || "default")
+
+    if (admin) return { role: "Admin", client }
+
+    try {
+        await client.connection.sendCommand(["GRAPH.QUERY"])
```

Comment:
this will alway be error there are missing arguments

Reply:
what args ?
i see that the send Command accept args (required) and options  (optional)